core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -153,11 +158,15 @@ public LockResource lockShared() throws InterruptedException {
    */
   public LockResource lockExclusive(StateLockOptions lockOptions)
       throws TimeoutException, InterruptedException {
+    LOG.debug("Thread-{} entered lockExclusive().", ThreadUtils.getCurrentThreadIdentifier());
     // Run the grace cycle.
     StateLockOptions.GraceMode graceMode = lockOptions.getGraceMode();
     boolean lockAcquired = false;
     long deadlineMs = System.currentTimeMillis() + lockOptions.getGraceCycleTimeoutMs();
     while (System.currentTimeMillis() < deadlineMs) {
+      LOG.info("Thread-{} entered grace-cycle of try-sleep: {}ms-{}ms for the total of {}ms",
```

Comment:
This will print every 10 minutes for 12 hours? Is that too noisy?

Reply:
made it one time.